assets/src/containers/App.js

```diff
@@ -49,7 +49,7 @@ function App (props) {
               render={props => <GradeDistribution {...props} disabled={!courseInfo.course_view_options.gd}
                 courseId={courseId} />} />
             <Route path='/:courseId/assignment'
```

Comment:
This commit handles disabling the view and other error cases. On to the files view now

Reply:
Nice this looks good